coordinator/src/services/translation/prescription/organization.ts

```diff
@@ -48,10 +56,29 @@ function isNhsTrust(fhirOrganization: fhir.Organization) {
   return organizationTypeCoding?.code === NHS_TRUST_CODE
 }
 
+function isDirectReference<T extends Resource>(
+  reference: fhir.Reference<T> | fhir.IdentifierReference<T>
+): reference is fhir.Reference<T> {
+  return !!(reference as fhir.Reference<T>).reference
+}
+
 function convertHealthCareProviderLicense(fhirOrganization: fhir.Organization, fhirBundle: fhir.Bundle) {
-  const fhirParentOrganization = fhirOrganization.partOf
-    ? resolveReference(fhirBundle, fhirOrganization.partOf)
-    : fhirOrganization
+  let fhirParentOrganization = fhirOrganization
+  const partOf = fhirOrganization.partOf
+  if (partOf) {
+    if (isDirectReference(partOf)) {
+      fhirParentOrganization = resolveReference(fhirBundle, partOf)
+    } else {
+      //TODO - Fix error FHIR paths for this case?
```

Comment:
does this still need doing?

Reply:
Would be nice :shrug: Basically if we don't, the errors will come back with a path of e.g. `Organization.identifier` instead of `Organization.partOf.identifier`

Reply:
But it was kinda annoying to implement